src/ingestion/sources/base.py

```diff
@@ -20,6 +20,9 @@ class SourceConfiguration:
     def validate(cls, configuration: str):
         raise NotImplementedError
 
+    def _connection_string_prefix(self):
+        raise NotImplementedError
```

Comment:
Don't think this needs to be a method that is required to implement the class

Reply:
removed from base